### ISSUE TITLE
Error with Simple Buttons

### ISSUE THREAD

**Issue by drehimself** [tailwindcss/plugin-examples#1]
Hello! Just messing around with the plugins in Laravel/Laravel Mix.

I can get css grid and object fit plugins working but can't get simple buttons working.

```
ERROR in ./resources/assets/css/main.css
Module build failed: ModuleBuildError: Module build failed: TypeError: Color(...).darken(...).hex is not a function at _.map (/Users/Dre/code/test-tailwind5/plugins/simple-buttons/index.js:114:114)
```
`tailwind.js` - same as the one in this repo

```
plugins: [
    require('tailwindcss/plugins/container')({
      // center: true,
      // padding: '1rem',
    }),
    require('./plugins/object-fit')(['responsive']),
    require('./plugins/simple-buttons')(),
    require('./plugins/css-grid')({
      grids: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12],
      gaps: {
        '0': '0',
        '4': '1rem',
        '8': '2rem',
      }
    })
  ],
```

**Comment by randyrankin** [comment]
It looks like you are missing the color package dependency. Add `"color": "^3.0.0"` to your package.json and run `npm update`, or run `npm install color`.

**Comment by adamwathan** [comment]
Can you share your repo with the lock file so I can try it out with the same package versions? That color package is in the package.json for this project and I can clone it and run it fresh here no problem so not sure what the issue is 🤔

**Comment by randyrankin** [comment]
I initially pulled in the package per the readme and all worked flawlessly (thank you @adamwathan!).  I then tried to implement the buttons plugin into an existing Laravel / Tailwind project and received the same error as @drehimself. After reviewing the `package.json` file from this package, I realized I was missing the colors package and, after I pulled that in, everything was fine. So, my suspicion is that @drehimself might be doing the same thing. Maybe not, but that was my experience.
